src/knd_class.json.c

```diff
@@ -179,9 +179,9 @@ static int export_conc_elem_JSON(void *obj,
 
     c->depth = 0;
     c->max_depth = 0;
-    if (self->max_depth) {
+    /*if (self->max_depth) {
```

Comment:
May be it should be deleted?
Why c->max_depth would be changed in export at all?

Reply:
The caller might wish to export objects with a desired depth of expansion, eg. expand the children of each class or grandchildren etc. The issue will be solved after implementing the concurrently accessed fields in kndClass.